app/index.html

```diff
@@ -22,6 +22,7 @@
   <link rel="stylesheet" href="bower_components/chosen/chosen.css" />
   <link rel="stylesheet" href="bower_components/angular-ui-tree/dist/angular-ui-tree.css" />
   <link rel="stylesheet" href="bower_components/angular-ui-select/dist/select.css" />
+  <link rel="stylesheet" href="../bower_components/angular-xeditable/dist/css/xeditable.min.css" />
```

Comment:
Incorrect link to  bower_components directory. Should be 'bower_components/' instead of '../bower_components/'